sdk/keyvault/azure-security-keyvault-secrets/test/ut/secret_client_test.cpp

```diff
@@ -1,15 +1,43 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
+#include "gtest/gtest.h"
 
-#include <gtest/gtest.h>
```

Comment:
Rever this, 
We use <> for external references and "" for references within the project.